docs/distributions/fedora/installation.md

```diff
@@ -0,0 +1,47 @@
+# Download the latest safe release
+
+Thanks so Mike for building. Download here. [https://github.com/mikeeq/mbp-fedora](https://github.com/mikeeq/mbp-fedora)
+
+# Hardware Requirements
+
+-   USB-C to Ethernet adapter
+    -   This isn't required, and you can use a USB WiFi Adapter instead.
+-   USB-C to USB adapter
+
+# Install Procedure
+
+(These instructions are reused from the Manjaro installation guide)
```

Comment:
In future we probably should consolidate these sections that are common to multiple distros (i.e. turn off secure boot, flash iso to usb) into one place and link to that from each distro's article, so we don't have to repeat ourselves and update more articles when things change. (But I don't think that needs to be done in this pr)

Reply:
Good call.